view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+						{{/if}}
+						<td>
+						{{if $u.is_deletable}}
+							<input type="checkbox" class="users_ckbx" id="id_user_{{$u.uid}}" name="user[]" value="{{$u.uid}}"/></td>
+						{{else}}
+							&nbsp;
+						{{/if}}
+						<td class="text-right">
+							<a href="#" onclick="return details({{$u.uid}})"><i class="fa fa-bars" aria-hidden="true"></i>
+						</td>
+					</tr>
+					<tr id="user-{{$u.uid}}-detail" class="hidden details">
+						<td>&nbsp;</td>
+						<td colspan="4">
+							{{if $order_users != $th_users.2.1}}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.2.1}}">
+									&#8597; {{$th_users.2.0}}</a> : {{$u.register_date}}</p>
+							{{/if}}
+
+							{{if $order_users != $th_users.3.1}}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.3.1}}">
+										&#8597; {{$th_users.3.0}}</a> : {{$u.login_date}}</p>
+							{{/if}}
+
+							{{if $order_users != $th_users.4.1}}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.4.1}}">
+										&#8597; {{$th_users.4.0}}</a> : {{$u.lastitem_date}}</p>
+							{{/if}}
+
+							{{if in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1]) }}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.5.1}}">
+										&#8597; {{$th_users.5.0}}</a> : {{$u.page_flags}} {{if $u.is_admin}}({{$siteadmin}}){{/if}} {{if $u.account_expired}}({{$accountexpired}}){{/if}}</p>
+							{{/if}}
+
+						</td>
+						<td class="text-right">
+							{{if $u.is_deletable}}
+								<a href="{{$baseurl}}/admin/users/block/{{$u.uid}}?t={{$form_security_token}}" title='{{if $u.blocked}}{{$unblock}}{{else}}{{$block}}{{/if}}'>
+									{{if $u.blocked==0}}
+									<i class="fa fa-ban" aria-hidden="true"></i>
+									{{else}}
+									<i class="fa fa-circle-o" aria-hidden="true"></i>
+									{{/if}}
+								</a>
+								<a href="{{$baseurl}}/admin/users/delete/{{$u.uid}}?t={{$form_security_token}}" title='{{$delete}}' onclick="return confirm_delete('{{$u.name}}')"><i class="fa fa-trash" aria-hidden="true"></i></a>
+							{{else}}
+								&nbsp;
+							{{/if}}
+						</td>
+					</tr>
+				{{/foreach}}
+				</tbody>
+			</table>
+			<div class="panel-footer text-right">
+						<button type="submit" name="page_users_block" class="btn btn-warning">	<i class="fa fa-ban" aria-hidden="true"></i> {{$block}} / <i class="fa fa-circle-o" aria-hidden="true"></i> {{$unblock}}</button>
+						<button type="submit" name="page_users_delete" class="btn btn-danger" onclick="return confirm_delete_multi()"><i class="fa fa-trash" aria-hidden="true"></i> {{$delete}}</button>
+			</div>
+		{{else}}
+			<div class="panel-body text-center bg-danger">NO USERS?!?</div>
+		{{/if}}
+		</div>
+
+
+
+	</form>
+
+
+
+
+
+<!--
```

Comment:
Smarty comments are done with `{{*` and `*}}`
e.g.
`{{* DELETED Users table *}}`